functions/__sf_section_conda.fish

```diff
@@ -0,0 +1,36 @@
+#
+# Conda
+#
+# Current Conda version.
+
+function __sf_section_conda -d "Display current Conda version"
+    # ------------------------------------------------------------------------------
+    # Configuration
+    # ------------------------------------------------------------------------------
+
+    __sf_util_set_default SPACEFISH_CONDA_SHOW true
+    __sf_util_set_default SPACEFISH_CONDA_PREFIX $SPACEFISH_PROMPT_DEFAULT_PREFIX
+    __sf_util_set_default SPACEFISH_CONDA_SUFFIX $SPACEFISH_PROMPT_DEFAULT_SUFFIX
+    __sf_util_set_default SPACEFISH_CONDA_SYMBOL "🅒 "
+    __sf_util_set_default SPACEFISH_CONDA_COLOR blue
+
+    # ------------------------------------------------------------------------------
+    # Section
+    # ------------------------------------------------------------------------------
+
+    [ $SPACEFISH_CONDA_SHOW = false ]; and return
+
+    # Show Conda version only if conda is installed and CONDA_DEFAULT_ENV is set
+    if not type -q conda; \
+      or test -z "$CONDA_DEFAULT_ENV";
+	    return
+    end
+
+    set -l conda_version (conda -V | cut -d' ' -f 2)
```

Comment:
I would suggest using [`string split`](https://fishshell.com/docs/current/commands.html#string-split) instead of `cut`, as it is the "fish" way of doing this sort of thing.